frontend/src/components/ReportSelect/index.tsx

```diff
@@ -111,8 +112,16 @@ export const ReportSelect = () => {
     const reportUri = `${values.report_type}`;
     // This will need to change as more report types are added
     const url = `${baseUri}/${values.project.value}/${reportUri}`;
+    // Build querystring params
+    const querystringParams = new URLSearchParams();
+    for (const field in values) {
+      if (queryFields.includes(field)) {
+        querystringParams.append(field, values[field].value.toString());
+      }
+    }
     axios(url, {
       method: "GET",
```

Comment:
Just wondering if this is always going to make sense using GET (easier for testing) or eventually are we going to need a POST

Reply:
I think it's better to use a GET since a POST is supposed to result in a new resource being created according to REST. It might get messy if there are a lot of querystring properties being passed but I don't think any of the reports have more than 3 or 4 inputs.

Reply:
I don't know if i totally agree with this.

Reply:
but yes... i think GET is fine for now